acasclient/acasclient.py

```diff
@@ -1841,4 +1841,57 @@ def delete_lot(self, lot_corp_name):
         if resp.status_code == 500:
             return None
         resp.raise_for_status()
+        return resp.json()
+        
+    def reparent_lot(self, lot_corp_name, new_parent_corp_name, dry_run=True):
+        """Reparent a lot
+
+        Args:
+            lot_corp_name (str): Corp name of lot to reparent
+            new_parent_corp_name (str): Corp name of new parent
+            dry_run (bool): Whether to perform a dry run, default True
+
+        Returns:
+            A dict with. For example
```

Comment:
missing a word here. Should this say `A dict with dependencies`? or `A dict with information about expected changes`